internal/gengapic/gengrpc.go

```diff
@@ -0,0 +1,406 @@
+// Copyright (C) 2021  Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gengapic
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/golang/protobuf/proto"
```

Comment:
Could migrate to `google.golang.org/protobuf/proto`. But that could/should be its own PR.

Reply:
We should definitely start using the v2 module for all helper methods where possible, but migrating the plugin implementation to v2 will be a tall task so we may not be able to do everything. 

+1 to a separate PR though

Reply:
Agreed, that belongs in another PR.